examples/04_manipulating_images/plot_multiply_image.py

```diff
@@ -0,0 +1,21 @@
+"""
+Multiply image
+==============
+
+Here we multiply the voxel values of an image by -1.
+"""
+
+from nilearn import datasets, plotting, image
+
+data = datasets.fetch_atlas_aal()
```

Comment:
I addressed all other comments except this one. I recognize that computing the negate of an atlas is somewhat weird but is it meaningful to do the same with a stat map ?
I could also add another example with a more realistic use case.